packages/fxa-payments-server/src/routes/Product/index.tsx

```diff
@@ -221,7 +221,12 @@ export const Product = ({
     );
   }
 
-  if (!plans.result || plans.error !== null || !selectedPlan) {
+  if (
+    !plans.result ||
+    plans.error !== null ||
+    !selectedPlan ||
+    !selectedPlan.active
```

Comment:
Q: could `selectedPlan` be null/undefined, or is there a risk that `selectedPlan.active` would throw an error if it's trying to read `undefined.active` here?